frontend/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "npm-run-all --print-label 11ty:clean --parallel 11ty:watch parcel:serve",
-    "build": "npm-run-all --print-label 11ty:clean 11ty:build parcel:build",
+    "start": "cross-env NODE_ENV=development npm-run-all --print-label 11ty:clean --parallel 11ty:watch parcel:serve",
+    "build": "cross-env NODE_ENV=production npm-run-all --print-label 11ty:clean 11ty:build parcel:build",
```

Comment:
Let's avoid setting this variable here. `.env` file will be used only in development, on the deployment server all the variables will be set in another way. For development it is better to fallback to `development` in the code.

Reply:
`CROSS_ENV` package removed, we are using `.env.local` for `development`